### ISSUE TITLE
timeout while running long commands

### ISSUE THREAD

**Issue by eplaut** [tomerfiliba/rpyc#169]
Hi,

Running long commands (>30 secs) raise socket.timeout exception

test case:

```
$ rpyc_classic.py &
INFO:SLAVE/18812:server started on [0.0.0.0]:18812

$ python -c 'import rpyc; c = rpyc.classic.connect("localhost"); c.modules.time.sleep(40)'
INFO:SLAVE/18812:accepted 127.0.0.1:43377
INFO:SLAVE/18812:welcome [127.0.0.1]:43377
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/eli/qa/.venv/local/lib/python2.7/site-packages/rpyc/core/netref.py", line 196, in __call__
    return syncreq(_self, consts.HANDLE_CALL, args, kwargs)
  File "/home/eli/qa/.venv/local/lib/python2.7/site-packages/rpyc/core/netref.py", line 71, in syncreq
    return conn.sync_request(handler, oid, *args)
  File "/home/eli/qa/.venv/local/lib/python2.7/site-packages/rpyc/core/protocol.py", line 464, in sync_request
    raise socket.timeout
socket.timeout
INFO:SLAVE/18812:goodbye [127.0.0.1]:43402
```

rpyc/core/protocol.py: Connection.SYNC_REQUEST_TIMEOUT = 30

Thanks

**Comment by tomerfiliba** [comment]
i see this was introduced by https://github.com/tomerfiliba/rpyc/commit/7abaf580fecd0d19ce1383559d1cf41c4688bf18 ... for once you can change ``c.SYNC_REQUEST_TIMEOUT`` to your liking, but it's certainly a bug introduced by this commit.

**Comment by coldfix** [comment]
Was fixed by a00be891a1f9db116f536b543110048af329222c.
